### PR TITLE
{math}[GCC/13.2.0] SharedMeatAxe v1.0.1

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/SharedMeatAxe/SharedMeatAxe-1.0.1-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    'zef', 'ziv', 'zmu', 'zor', 'zpt', 'zsc', 'zsy', 'ztr', 'zvp', 'decomp', 'mkdotl', 'mkhom_old', 'mktree',
+    'pseudochop', 'soc', 'tuc', 'zcf', 'zct', 'zev', 'zkd', 'zmw', 'zpo', 'zqt', 'zsi', 'ztc', 'zts',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % bin for bin in local_bins] + ['include/meataxe.h', 'lib/libmtx.%s' % SHLIB_EXT],
+    'dirs': []
+}
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/s/SharedMeatAxe/SharedMeatAxe-1.0.1-GCC-13.2.0.eb:36]: strip out duplicate empty line:
```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3123.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/529fce612c064794420292a033f8957f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**